core/frontend/src/tools/Tool.ts

```diff
@@ -394,6 +394,7 @@ export class Tool {
 
   private static getLocalizedKey(name: string): string | undefined {
     const key = `tools.${this.toolId}.${name}`;
+    // eslint-disable-next-line deprecation/deprecation
     const val = IModelApp.localization.getLocalizedStringWithNamespace(this.namespace, key);

```

Comment:
Replacing this call with one of the suggestions in your `@deprecated` message isn't an option?

Reply:
I can absolutely do that for each place that I added the eslint-disabled line and get the changes over with now. I was unsure whether to leave this for later or do it now.

Reply:
Yeah, generally if we tell people to stop using an API we should stop using it too.